backend/lambda/lambdaRetrieveDataLibrary/lambda_return_data.py

```diff
@@ -31,10 +30,9 @@ def getCredentials() -> db_creds:
     )
 
 
-def lambda_handler(event, context):
-  
+def check_table(table):
     try:
-        credential = getCredentials()
+        credential = get_postgres_credentials(secret_Name='secretName') # replace the secret with the actual secret for the DB creds
```

Comment:
secret name should be taken from environment variable as we are in a lambda environment.
`os.getenv("secret_name")`